appendix_modifer.md

```diff
@@ -46,11 +58,12 @@ The following document defines the standardized modifiers that can be used in Si
 * `wide`: Alias for `utf16le` modifier
 * `utf16`: Prepends a [byte order mark](https://en.wikipedia.org/wiki/Byte_order_mark) and encodes UTF16, e.g. `cmd` > `FF FE 63 00 6d 00 64 00` (only used in combination with base64 modifiers)
 
-* `windash`: creates all possible permutations of the `-` and `/` characters. Windows command line flags can often be indicated by both characters. Using the `windash` modifier converts `-` values into `/` and vice versa and uses all possible permutation of strings in the selection.
+
+# Specific
 
 * `expand`: Modifier for expansion of placeholders in values. The final behavior of the replacement is determined by processing pipeline transformations. Current possibilities in pySigma are:
   * Expand to value list (`ValueListPlaceholderTransformation`/`value_placeholders`)
   * Replace with query expression in target query language (`QueryExpressionPlaceholderTransformation`/`query_expression_placeholders`)
   * Replace placeholder with wildcard `*`, which should only be used as last resort. (`WildcardPlaceholderTransformation`/`wildcard_placeholders`)
 
-
+* `fieldref`: Modifier a plain string into the field reference type.

```

Comment:
```suggestion
* `fieldref`: Modifies a plain string into the field reference type.
```
First time I am hearing this modifier. Can we give an example? As is, I don't understand what it does.